contracts/OpenSTUtils.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.4.23;
+
+import "./BytesLib.sol";
+import "./MerklePatriciaProof.sol";
+import "./RLPEncode.sol";
+
+
+library OpenSTUtils {
```

Comment:
Name is going to be confusing: as we have OpenSTUtility and now OpenSTUtils. One for utility chain, one for keeping utility functions ...

Reply:
For now its renamed to `OpenSTHelper` with a comment that this will become Utils.sol